azure-pipelines.yml

```diff
@@ -23,24 +26,89 @@ pool:
 # these group variables can be accessed as $(variable_name)
 variables:
   - group: hwsc-dev-container-vars # name of variable group file that contains docker hub user/email/image-names
+  - name: GOBIN # Go binaries path
+    value: '$(GOPATH)/bin'
+  - name: GOROOT  # Go installation path
+    value: '/usr/local/go1.11'
+  - name: GOPATH # Go workspace path
+    value: '$(system.defaultWorkingDirectory)/gopath'
+  - name: modulePath # Path to the module's code
+    value: '$(GOPATH)/src/github.com/$(build.repository.name)'
 
 # task: DownloadSecureFile@1
 # download a Secure File to a temporary location on build or release agent
 # Secure Files is referenced from Azure DevOps > Pipelines > Library > "Secure Files" tab
 # once downloaded, the Secure File is located in $(Agent.TempDirectory) directory of Azure Pipelines Agent
 # selected secure files are deleted after the build or release is finished
 steps:
+  - script: printenv
+    displayName: 'Print Environment Variables'
+
+  - script: |
+      mkdir -p '$(GOBIN)'
+      mkdir -p '$(GOPATH)/pkg'
+      mkdir -p '$(modulePath)'
+      shopt -s extglob
+      mv !(gopath) '$(modulePath)'
+      echo '##vso[task.prependpath]$(GOBIN)'
+      echo '##vso[task.prependpath]$(GOROOT)/bin'
+    displayName: 'Set up the Go workspace'
+
+  - script: go get -v -t -d ./...
+    workingDirectory: '$(modulePath)'
+    displayName: 'Go Get Dependencies'
+
+  - script: |
+      export hosts_postgres_host="localhost"
+      export hosts_postgres_db="test_user_svc"
+      export hosts_postgres_user="postgres"
+      export hosts_postgres_password="secret"
+      export hosts_postgres_sslmode="disable"
+      export hosts_postgres_port="5432"
+      export hosts_smtp_host="smtp.gmail.com"
+      export hosts_smtp_port="587"
+      export hosts_smtp_username="hwsc.test@gmail.com"
+      export hosts_smtp_password="ftbsflgbewdpzzmo"
```

Comment:
Make this is as an environment variable in in DevOps pipeline if this is not safe.